spyder/workers/updates.py

```diff
@@ -27,16 +27,21 @@
 # Logger setup
 logger = logging.getLogger(__name__)
 
+CONNECT_ERROR_MSG = _(
+    'Unable to connect to the Spyder update service.'
+    '<br><br>Make sure the connection is working properly.'
```

Comment:
```suggestion
    '<br><br>Make sure your connection is working properly.'
```

I think this makes more sense and follows the same wording of the HTTP error message.